packages/demos/src/get-pkg-version.ts

```diff
@@ -0,0 +1,12 @@
+import path from 'path';
+import pkgDir from 'pkg-dir';
```

Comment:
I don’t think we need a this lib for this function. `require.resolve` should be sufficient.

Reply:
nope, unfortunately not

Reply:
Ah I see, because the resolved main file of a package can be anywhere in this package, we need to search up from there for the package.json. Correct?

Reply:
yes

Reply:
Isn’t [find-up](
https://www.npmjs.com/package/find-up) the better fitted lib then?

Reply:
It could also be used. But it doesn't matter in the end.